src/webextension/list/components/item-fields.js

```diff
@@ -16,11 +16,15 @@ import TextArea from "../../widgets/text-area";
 
 import styles from "./item-fields.css";
 
+import * as telemetry from "../../telemetry";
+
 const PASSWORD_DOT = "\u25cf";
 
 function CopyToClipboardButton({text, ...props}) {
   return (
-    <CopyToClipboard text={text}>
+    <CopyToClipboard text={text} onCopy={() => {
+      telemetry.recordEvent("itemCopied", "entryDetails");
```

Comment:
Two questions and a comment:

1. Do we want to distinguish between copying the username and copying the password? That might give us useful info.
2. Do we want to (eventually) distinguish between doing this in the management UI and the doorhanger? `entryDetails` doesn't seem especially useful at the moment, since seeing the copy button automatically means you're looking at an entry's details.
3. Finally, if we do use `entryDetails`, let's rename it to `itemDetails`. Internally, we call these things "items", but the UI calls them "entries" (this is slightly confusing, but I chose "item" to match the datastore's API, and the user-facing strings could really be anything).

Reply:
1. Yes, that would be ideal. Just couldn't quite figure the best way to implement that, so I did this as a quick first solution. If you could give me a quick pointer on where might be best to implement these as distinct events, I could do it before this is merged.

2. That's the plan for the next iteration, once I get my head around the doorhanger changes. no need to do it here I think. If we keep the "itemCopied" nomenclature for the event method, then we can add the changes to the event object in the next version to distinguish between the two and we should be fine w/r/t data continuity. 

3. I'm fine with renaming entryDetails to itemDetails for this PR. I'll do it

Reply:
For (1), easiest would be something like this:

```js
function CopyToClipboardButton({text, field, ...props}) {
  return (
    <CopyToClipboard text={text} onCopy={() => {
      telemetry.recordEvent(`${field}Copied`, "entryDetails");
    }}>
      <Button {...props}/>
    </CopyToClipboard>
  );
}

CopyToClipboardButton.propTypes = {
  text: PropTypes.string.isRequired,
  field: PropTypes.oneOf(["username", "password"]).isRequired,
};
```

Reply:
Oh, and I forgot, you'd have to use `<CopyToClipboardButton text={blah} field="username"/>`